### ISSUE TITLE
If there are two config files with same name in different folders, one of them gets ignored.

### ISSUE THREAD

**Issue by larsbs** [balderdashy/sails#3171]
Sails version: 0.11.0

**Description**
If I put two config files in **./config/** with the same name but in different subfolders, sails only load in `sails.config` one of them.

**Example**
I usually put my routes in a **./config/routes/** subfolder instead of the **./config/route.js** file. I usually have a **./config/fixtures/** folder too. If there are files with the same name in these two folders, one of them gets ignored. For instance:

*./config/routes/user.js*
```javascript
module.exports.routes = {
  '/profile': 'UserController.token'
};
```

*./config/fixtures/user.js*
```javascript
module.exports.fixtures = {
  User: [
    {
      id: 1,
      name: 'Lars'
    }
  ]
};
```
In this scenario, `sails.config.routes` is populated properly, but `sails.config.fixtures` appears empty.

**Proposed solution**
I don't know so much about sails.js core, but maybe checking the full path of the file instead of only the filename to test if the file has been already loaded.

**Comment by sailsbot** [comment]
Thanks for posting, @larsbs.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
